<?xml version="1.0"?>
<launch>

<!-- Launching the Moveit configuration -->  
<include file="$(find ur10_moveit_config)/launch/ur10_moveit_planning_execution.launch">
<arg name = "sim" value="true"/>
</include>

<node pkg = "pick_and_place" name = "basic_pick_place" type="pick_and_place_node" output = "screen">
<param name = "joint1" value = "3.1" />
<param name = "joint2" value = "-1.5" /> <!-- -1.3-->
<param name = "joint3" value = "2.4" />  <!-- 2.4 -->	
<param name = "joint4" value = "3.75" />  <!-- -1.57 -->
<param name = "joint5" value = "4.7" />   <!-- 1.57 -->
<param name = "joint6" value = "0" />

<!--param name = "z_offset" value = "0.02" />
<param name = "x_test" value = "-0.233" />
<param name = "y_test" value = "-0.6" />
<param name = "z_test" value = "0.723"-->

<!--param name = "x_test" value = "-0.1" />
<param name = "y_test" value = "0.164" />
<param name = "z_test" value = "0.724"-->

<!--param name = "x_test" value = "-0.5" />
<param name = "y_test" value = "0.03" />
<param name = "z_test" value = "0.724"-->

<param name = "x_test" value = "-0.5" />
<param name = "y_test" value = "-0.735" />
<param name = "z_test" value = "0.724"/>

<param name="tray_length" value="0.2" />
</node>

</launch>
